packages-common/pluggable-widgets-tools/utils/web/common.ts

```diff
@@ -0,0 +1,14 @@
+export function parseStyle(style = ""): { [key: string]: string } {
```

Comment:
`parseInlineStyle`? Plus the same about documentation.

I also think that this method deserves some unit tests...

Reply:
parseInlineStyles

Reply:
Renamed it to `parseInlineStyle`, because we are just receiving one style here, whereas with `mergeNativeStyles` we are dealing with multiple style definitions.